app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +14,27 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.CoachMe"
         tools:targetApi="31">
+
+        <!--
+             Before you run your application, you need a Google Maps API key.
+             To get one, follow the directions here:
+                https://developers.google.com/maps/documentation/android-sdk/get-api-key
+             Once you have your API key (it starts with "AIza"), define a new property in your
+             project's local.properties file (e.g. MAPS_API_KEY=Aiza...), and replace the
+             "YOUR_API_KEY" string in this file with "${MAPS_API_KEY}".
+        -->
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyCxeadFPFfBmwiZD9sUVepScMaYNtKNvFw" />
+            <!-- MAPS_API_KEY is ${MAPS_API_KEY}  -->
+        <meta-data
```

Comment:
It seems that we already have the Google Maps API key. It might reduce confusion in in the future to remove the two commented out parts so that we don't create a second one.